docs/source/dataframe-create.rst

```diff
@@ -57,23 +59,61 @@ Pandas:
 Creating
 --------
 
-Reading from various locations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Read from CSV
+~~~~~~~~~~~~~
 
-For text, CSV, and Apache Parquet formats, data can come from local disk,
-the Hadoop File System, S3FS, or other sources, by prepending the filenames with
-a protocol:
+You can use :func:`read_csv` to read one or more CSV files into a Dask DataFrame.
+It supports loading multiple files at once:
+  
+.. code-block:: python
+
+   >>> df = dd.read_csv('myfiles.*.csv')
+
+Or you can break up a single large file with the ``blocksize`` parameter:
+
+.. code-block:: python
+
+   >>> df = dd.read_csv('largefile.csv', blocksize=25e6)  # 25MB chunks  
+
+Changing the ``blocksize`` parameter will change the number of partitions (see the explanation on
+:ref:`partitions <dataframe-design-partitions>`). A good rule of thumb when working with
+Dask DataFrames is to keep your partitions under 100MB in size.
+
+Read from Parquet
+~~~~~~~~~~~~~~~~~
+
+Similarly, you can use :func:`read_parquet` for reading one or more Parquet files.
+You can read in a single Parquet file:
+
+.. code-block:: python
+
+   >>> df = dd.read_parquet("path/to/mydata.parquet")
+
+Or multiple Parquet files:
 
 .. code-block:: python
 
-   >>> df = dd.read_csv('my-data-*.csv')
-   >>> df = dd.read_csv('hdfs:///path/to/my-data-*.csv')
-   >>> df = dd.read_csv('s3://bucket-name/my-data-*.csv')
+   >>> df = dd.read_parquet("path/to/my/parquet/")
```

Comment:
It's not clear to (naive user) me how this points to multiple Parquet files, since the CSV example used the `*`. Can this be clarified? On a related note, Dask usually writes a folder called `dataset.parquet` which contains files called `part.0.parquet`, which may also be confusing here.

Reply:
good point! I added that this is a directory of parquet files